coalib/settings/ConfigurationGathering.py

```diff
@@ -266,7 +267,7 @@ def load_configuration(arg_list,
             save = True
 
         coafile_sections = load_config_file(config,
-                                            silent=save or silent)
+                                            silent=save)
```

Comment:
Just force silent to true as per https://github.com/coala/coala-quickstart/issues/62#issuecomment-289267092. this may break tests though which may need modification.